sql/sqlcheck/sqlcheck.go

```diff
@@ -130,6 +131,45 @@ var Destructive = &driverAware{
 	},
 }
 
+// These are the regexes used for detecting renames in statements for StatementRename.
+var (
+	mysqlRenameRegex = regexp.MustCompile("ALTER TABLE ([`\\w]+) RENAME COLUMN ([`\\w]+) TO ([`\\w]+)")
+	mysqlChangeRegex = regexp.MustCompile("ALTER TABLE ([`\\w]+) CHANGE(?: COLUMN)? ([`\\w]+) ([`\\w]+) [\\w()]+")
+)
```

Comment:
Hey, put this logic in the drivers, please. See the "Register" example in the test file.